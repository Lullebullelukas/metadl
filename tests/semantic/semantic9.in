A(_).
A(_) :- A(1).
A(x) :- B(x), EQ(x, _).