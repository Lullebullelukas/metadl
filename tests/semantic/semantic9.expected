_ is not allowed in rule heads or facts.
_ is not allowed inside constraints.
